numpy/array_api/_typing.py

```diff
@@ -37,7 +37,7 @@
 
 Device = Any
 Dtype = Type[
-    Union[[int8, int16, int32, int64, uint8, uint16, uint32, uint64, float32, float64]]
+    Union[tuple([int8, int16, int32, int64, uint8, uint16, uint32, uint64, float32, float64])]
```

Comment:
```suggestion
    Union[int8, int16, int32, int64, uint8, uint16, uint32, uint64, float32, float64]
```
Removing the surrounding list entirely should remove the need for converting it into a tuple.